tests/1-3-conformance.js

```diff
@@ -23,6 +27,18 @@ describe('Basic Conformance', function() {
     const endpoints = new TestEndpoints({implementation, tag});
 
     describe(name, function() {
+      let issuedVc;
+      before(async function() {
+        try {
+          issuedVc = await endpoints.issue(require(
+            './input/credential-ok.json'));
+        } catch(e) {
+          console.error(
+            `Issuer: ${name} failed to issue "credential-ok.json".`,
```

Comment:
we should probably store this error and ensure it shows up in the report so they know that the issuance failed.